pypeit/scripts/coadd_2dspec.py

```diff
@@ -152,14 +153,31 @@ def main(args):
         all_spec2d['meta']['bkg_redux'] = bkg_redux
         all_spec2d['meta']['find_negative'] = find_negative
 
+        # get only_slits and exclude_slits if they are set
+        only_dets, only_spat_ids, exclude_dets, exclude_spat_ids = None, None, None, None
+        if par['coadd2d']['only_slits'] is not None and par['coadd2d']['exclude_slits'] is not None:
+            msgs.warn('Both `--only_slits` and `--exclude_slits` are set. They are mutually exclusive. '
+                      'Using `--only_slits` and ignoring `--exclude_slits`')
+            par['coadd2d']['exclude_slits'] = None
```

Comment:
I'm probably being a broken record, but should we be changing the parameter set, or using a new variable, similar to my suggestion of not changing the function input value?  E.g., can we leave the parameter as is and instead set `exclude_dets = None` and `exclude_spat_ids = None`?

Reply:
Thanks @kbwestfall! I re-organized this `if` sequence because it wasn't actually working if both  `par['coadd2d']['only_slits']`  and `par['coadd2d']['exclude_slits']` are not `None`. And I removed `par['coadd2d']['exclude_slits'] = None`